sem/database.py

```diff
@@ -329,16 +366,27 @@ def get_results(self, params=None, result_id=None):
             else:
                 query_params[key] = params[key]
 
-        # Handle case where query params has no keys
+        # Handle case where query params has no keys                                                            #TODO - How will this case occur?
```

Comment:
This might happen if the user passes `{}` as `params`. In this case, we match all results!

Reply:
Understood. Thanks!